packages/pwa-kit-extension-sdk/package.json

```diff
@@ -42,6 +42,7 @@
     "fs-extra": "^11.2.0",
     "handlebars": "^4.7.8",
     "hoist-non-react-statics": "^3.3.2",
+    "lodash.merge": "^4.6.2",
```

Comment:
I thought that there was another merge lib that you talked about using?

Reply:
Yeah, there was another package called `deepmerge` that we've used. But a few things I don't like about it:
- it's heavier than `lodash.merge`
- the default behaviour of the array treatment is different than what I expected, so we'll need to pass in an argument to customize it

The `lodash.merge` is a smaller package, and it's already been approved so we don't need to file 3PP.